docs/UserGuide.md

```diff
@@ -127,7 +129,7 @@ Examples:
 
 ### Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified person from the aSectresBook.
```

Comment:
Sorry, just saw this.
Change `aSectresBook` to `SectresBook`

Reply:
Yikes I didnt see that either, updated and pushed